helios-tools/src/main/java/com/spotify/helios/cli/command/ControlCommand.java

```diff
@@ -118,7 +118,7 @@ private boolean run(final Namespace options, final Target target, final PrintStr
                       final BufferedReader stdin)
       throws InterruptedException, IOException {
 
-    final HeliosClient client = Utils.getClient(target, err, username);
+    final HeliosClient client = Utils.getClient(target, err, username, options);
```

Comment:
I care little about this, but... seems like the current pattern is to pass global options as parameters to the `run()` method (e.g. `username`, `json`). why not just follow that convention? or change it around so everything is accessed via `options`?